package.json

```diff
@@ -1,22 +1,30 @@
 {
   "name": "@peak-ai/ais-service-discovery",
-  "version": "0.2.0",
-  "main": "src/index.js",
+  "version": "0.2.1",
+  "main": "lib/index.js",
   "repository": "git@github.com:peak-ai/ais-service-discovery-js.git",
-  "author": "<ewan.valentine@peak.ai>",
+  "author": "<engineering@peak.ai>",
   "license": "MIT",
   "scripts": {
     "test": " jest ./src/**",
-    "lint": "eslint ./src"
+    "lint": "eslint ./src",
+    "start": "npm run build:live",
+    "build": "tsc -p .",
+    "build:live": "nodemon --watch 'src/**/*.ts' --exec \"ts-node\" src/index.ts"
```

Comment:
It would be good to lift prettier from one of the other repos into here to fix up some differences in consistency around the project, but if it looks like it's a bigger job then don't worry about it.